src/trade-params/TradeParams.js

```diff
@@ -44,6 +46,18 @@ import { changeAsset, changeBarrier1, changeBarrier2, changeCategory, changeStar
  * default may not be a good idea at all as client might always want to input value
  */
 
+const errorToShow = errorObj => {
+    const { barrierError, contractError, durationError, proposalError, purchaseError } = errorObj;
+
+    if (contractError) return contractError;
+    if (barrierError) return barrierError;
+    if (durationError) return durationError;
+    if (proposalError) return proposalError;
+    if (purchaseError) return purchaseError;
+
+    return undefined;
```

Comment:
Also, do we care wha type of error is it?
Wouldn't a list of errors be enough?
Maybe not, just a question...